src/libcmd/command.cc

```diff
@@ -165,7 +165,7 @@ BuiltPathsCommand::BuiltPathsCommand(bool recursive)
     });
 }
 
-void BuiltPathsCommand::run(ref<Store> store)
+void BuiltPathsCommand::runIs(ref<Store> store, Installables && installables)
```

Comment:
`runIs` is not a great name. Maybe `runInstallables`? And similarly for `runBPs`, `runSPs`, `runI`, ...

Reply:
It's just `run` again :)